docs/aws-config.md

```diff
@@ -139,9 +139,9 @@ kubectl create secret generic cloud-credentials \
 
 Specify the following values in the example files:
 
-* In `examples/aws/00-ark-config.yaml`:
+* In `examples/aws/00-ark-config.yaml` and `examples/aws/05-ark-backupstoragelocation.yaml`:
```

Comment:
I like splitting this into 2 bullets like you did in `azure-config.md`, one for `Config` and one for `BackupStorageLocation`.  Then you can link to the Config definition doc for the config bullet, and the BSL definition doc for the BSL bullet.